packages/pass-style/src/types.js

```diff
@@ -74,7 +105,14 @@ export {};
  */
 
 /**
- * @typedef {Passable} RemotableObject
+ * @template {string} S pass style
+ * @template {InterfaceSpec} I interface tag
+ * @typedef {PassStyled<S> & {[Symbol.toStringTag]: I}} TaggedRecord
+ */
+
+/**
+ * @template {InterfaceSpec} [I=string]
+ * @typedef {TaggedRecord<'remotable', I>} RemotableObject
```

Comment:
Why RemotableObject rather than Remotable? What differentiates them?

Reply:
`Remotable` is a value 
https://github.com/endojs/endo/blob/master/packages/pass-style/src/make-far.js#L80C1-L82

I'll document